### PR TITLE
Milestone 2: User interface

### PR DESCRIPTION
**Pull request by arthurborgesdev**
My partner and I wrote a user interface with initial logic containing mock-up code for the Tic Tac Toe Game. Things have done:

- Prompt the user for names
- Assigned the users to a players array
- Print information to the screen stating who and when is going to play
- Draw the board in a box using tty-box gem and display it on the terminal
- Handle error of invalid input from not in the correct range (1-9)
- Mockup of the result of the match (Win and Tie)
- Run rubocop and fix linter errors

### CODE REVIEW

**Review by BrianSammit — CHANGES_REQUESTED** [review]
CHANGES REQUIRED :red_circle: :stop_sign: 
------------------
Hello, @arthurborgesdev @Maha-Magdy You did an amazing job so far in this milestone. :muscle: I really like the displaying game is really clean. :1st_place_medal: :100: 
However, there are a few changes to make to complete it :hammer_and_wrench: . see them below :eyes: :point_down: 
----------------
- [x] On your pull request description, you did not `drew` the code. You `wrote` the code. Please change this `drew` for `wrote`. 
- [x] You have minor grammar errors in the pull request description. Please fix it. Use Grammarly as a tool to checks these errors. 
- [x] When I try to run your project it throws me an error. 

Screenshot from 2021-03-29 15-30-10

To solve it just include this gem `gem 'tty-box'` inside your Gemfile. Remember that you must write in your Readme file that they must run `bundle` to install this gem. (You gonna work on the Readme file on the last milestone). 
- [x] Please validate the user's input so they cannot enter an empty name. 

Screenshot from 2021-03-29 15-36-51

- [x] You have some minor errors on the commit messages. 
- You have some misspelling errors. 
- You must use the imperative for the commits messages. 
[Here](https://github.com/trein/dev-best-practices/wiki/Git-Commit-Best-Practices#formatting-rules) You can read about the formatting rules for a good commit message. [Here](https://docs.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message) you can read about how to change a commit message. You don't have to make these changes this time. But keep these formating rules in mind for the next commits. 
--------------
Make these required changes :point_up_2: and sent it again for another review. :smile: 
Reachme on SLACK if you have any questions. 
@ Brian Sammit Cruz Rodriguez. 
HAPPY CODING!!! 
AngelicConcreteHypsilophodon-max-1mb

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'rubocop', '~>0.81.0', 'tty-box'
```
Comment (jcy2704) [inline Gemfile]: - When you add gems to the `Gemfile` you **don't add it with a comma.** Although it seems that is the best way to do it, Ruby doesn't work like that. You need to **add new gems in their respective line.**

**For example**
```
  gem 'rubocop'
  gem 'tty-box'
  # gem 'whatever-gem'
```

- [x] Separate new gems in a new line

**Review by jcy2704 — CHANGES_REQUESTED** [review]
# CHANGES REQUIRED 👀🤔

## Hey @arthurborgesdev and @Maha-Magdy!
### I really like your user interface!! I wouldn't even think to use TTY to create a box when I was working on this project 🤯. GOOD JOB! 👍

### You've also done a great job trying to fix the last changes 🙌 and you are really close to getting approved, but there is a slight error that you are still missing.

- **Your project still presents a problem with `tty-box`. I've run `bundle` in my terminal but I get the following error.**

Screen Shot 2021-03-29 at 6 47 08 PM

- **SEE BELOW FOR CHANGES 👇👇👇**

<hr>

### Keep the awesome work! 💪 Happy Coding for y'all!! 💻

### If you have any questions you can ask me on my Slack.

**Best,**

**[💁‍♂️@ Steven Jack Chung Yau](https://stevenchung.me)**

[LinkedIn](https://linkedin.com/in/stevenjchung) | [Twitter](https://twitter.com/_yiak) | [GitHub](https://github.com/jcy2704)

File: Gemfile (hunk 1 above)
Comment (Maha-Magdy) [reply Gemfile]: Thanks for your comment. It helps to understand.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Maha-Magdy @arthurborgesdev, good job on this milestone!! 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Well done team

Now you can merge this branch 🚀 

You can contact me on slack if you have any questions, just search for me as @ Miguel Ricaño.

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

**Pull request merged**